sqlglot/dialects/athena.py

```diff
@@ -98,41 +101,97 @@ class Athena(Trino):
       - Used for DML operations
         - SELECT, INSERT, UPDATE, DELETE, MERGE
 
-    The SQLGlot Athena dialect tries to identify which engine a query would be routed to and then uses the parser / generator for that engine
-    rather than trying to create a universal syntax that can handle both types.
+    The SQLGlot Athena dialect tries to identify which engine a query would be routed to and then
+    uses the parser / generator for that engine rather than trying to create a universal syntax that
+    can handle both types.
     """
 
     class Tokenizer(Trino.Tokenizer):
         """
         The Tokenizer is flexible enough to tokenize queries across both the Hive and Trino engines
         """
 
-        IDENTIFIERS = ['"', "`"]
-        KEYWORDS = {
+        IDENTIFIERS: ClassVar[list] = ['"', "`"]
+        KEYWORDS: ClassVar[dict] = {
```

Comment:
Why were these added? This pattern is inconsistent with the rest of the codebase.